fastlane/Fastfile

```
@@ -0,0 +1,84 @@
+      debug: true
+    )
+  end
+
+  desc "Build, Sign, and Distribute Release APK to Play Store"
+  lane :build_and_distribute_release do |options|
+    keystore_path = File.expand_path('keystore.jks')
+    gradle(
+          task: "bundle#{options[:variant]}Release --stacktrace",
+          properties: {
+            "android.injected.signing.store.file" => keystore_path,
+            "android.injected.signing.store.password" => "#{ENV['KEYSTORE_PASSWORD']}",
```

rubocop (1.69.1)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)